adds an option to the PR template for config files (#2705)
DataDog/integrations-core: .github/PULL_REQUEST_TEMPLATE.md

- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

> - [ ] If PR adds a configuration option, it should be added to the configuration file.

Reviewer: ```suggestion
- [ ] If PR adds a configuration option, it has been added to the configuration file.
```
